Source/Public/Get-VSTeamBillingToken.ps1

```diff
@@ -0,0 +1,22 @@
+function Get-VSTeamBillingToken {
+   [CmdletBinding(HelpUri = 'https://methodsandpractices.github.io/vsteam-docs/docs/modules/vsteam/commands/Get-VSTeamBillingToken')]
+   param ()
+
+   $sessionToken = @{
+      appId        = 00000000 - 0000 - 0000 - 0000 - 000000000000
+      force        = $false
+      tokenType    = 0
+      namedTokenId = "CommerceDeploymentProfile"
+   }
+
+   $billingToken = _callAPI `
+      -NoProject `
+      -method POST `
+      -ContentType "application/json" `
+      -area "WebPlatformAuth" `
+      -resource "SessionToken" `
+      -version '3.2-preview.1' `
```

Comment:
@DarqueWarrior  not sure which enumeration to use here. This is not a public API and on GUI this API version is being used. Do we need another type of API version? Because there is none for this and I would like to ensure that it's working properly.

Reply:
I we create one for this call it WebPlatformAuth. That way we are consistent.